pkg/monitortests/kubeapiserver/auditloganalyzer/handle_audit_violation.go

```diff
@@ -0,0 +1,78 @@
+package auditloganalyzer
+
+import (
+	"fmt"
+	"github.com/openshift/origin/pkg/test/ginkgo/junitapi"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	auditv1 "k8s.io/apiserver/pkg/apis/audit/v1"
+	"strings"
+	"sync"
+)
+
+func CheckForViolations() *violations {
+	return &violations{}
+}
+
+type violations struct {
```

Comment:
normally i'm biased toward shorter names but in this case... "violation" seems a little raw by itself. maybe `auditViolations` and similar for other names?